man/usr/share/man/man1m/piadm.1m.md

```diff
@@ -5,14 +5,15 @@ piadm(1M) -- Manage SmartOS Platform Images
 ## SYNOPSIS
     /usr/sbin/piadm [-v | -vv] <command> [command-specific arguments]
 
-    piadm activate|assign <PI-stamp> [ZFS-pool-name]
+    piadm activate|assign <PI-stamp/ipxe> [ZFS-pool-name]
     piadm avail
     piadm bootable
     piadm bootable [-dr] <ZFS-pool-name>
     piadm bootable -e [ -i <source> ] <ZFS-pool-name>
     piadm install <source> [ZFS-pool-name]
     piadm list [ZFS-pool-name]
     piadm remove <PI-stamp> [ZFS-pool-name]
+    piadm update ipxe [ZFS-pool-name]
```

Comment:
Reviewers: I'm contemplating just making this a single-keyword `update-ipxe`.

Reply:
It occurs to me that we maybe shouldn't have verbs for ipxe at all. Instead, piadm would be used to specify the "fallback" PI only, but the boot bits indiscriminately use the loader/ipxe provided by the HN that are already on disk. The more I think about it, I don't think we're ready to have CNs booting a local PI for normal operation.

We may also want Triton CNs to only be able to install platform images that are available from the headnode, and not look in manta. This is complicated by the fact that the HN doesn't retain the PI tarball after unpacking. Maybe this could be fixed by having sdcadm platform add the platform images to imgapi, then Triton CNs could fetch it from imgapi. By default, CNs won't have a default gateway to reach manta anyway. So the process for Triton would be something like this, executed from the headnode:

```
curl -o <some image>
sdc-oneachnode -c -g <image> -d /var/tmp
sdc-oneachnode -c 'piadm bootable -e /var/tmp/<image> zones'
```

Reply:
My previous comments about "prefer pipe" are kind of moot if we go this route.